hieradata_aws/class/production/jenkins.yaml

```diff
@@ -1,5 +1,6 @@
 ---
 govuk_jenkins::job_builder::jobs:
+  - govuk_jenkins::jobs::athena_fastly_logs_check
```

Comment:
Does `govuk_jenkins::jobs::athena_fastly_logs_check::databases` need to added to this jobs list?